README.rst

```diff
@@ -202,7 +251,24 @@ Updates the version and date of the most recent changelog entry.
     Arguments:
       TAG       Version number to set on the entry.
 
+Config file
```

Comment:
Please note that this pull request still lacks the tests and I've planned to add them, I just don't know yet when I'll find the time to do so. This todo is tracked by [this issue](https://github.com/dpsenner/chag/issues/1).

Note that I considered to parse a plain ini file but since that is by far not trivial I opted for sourcing the configuration file as bash.